va/va.go

```diff
@@ -623,9 +624,15 @@ func (va *ValidationAuthorityImpl) performLocalValidation(
 	return records, nil
 }
 
-// PerformValidation validates the challenge for the domain in the request.
-// The returned result will always contain a list of validation records, even
-// when it also contains a problem.
+// PerformValidation conducts a local Domain Control Validation (DCV) and CAA
+// check for the specified challenge and dnsName. When invoked on the primary
+// Validation Authority (VA) and the local validation succeeds, it also performs
+// DCV and CAA checks using the configured remote VAs. An error is only returned
+// in cases of internal error (e.g., network failure). When a requester fails a
+// DCV or CAA check, a corresponding problem is included in the ValidationResult.
+// The ValidationResult always includes a list of ValidationRecords, even when
+// it also contains a Problem. This method does NOT implement Multi-Perspective
+// Issuance Corroboration as defined in BRs Sections 3.2.2.9 and 5.4.1.
```

Comment:
Another comment-on-comment: network failure is actually one kind of failure type that isn't returned by `ValidationAuthorityImpl`. If the caller gets a network error (timeout, dial failure, etc), that's returned by gRPC itself.

I would express this something like "Failed validations are indicated by a non-nil `problems` in the returned ValidationResult. PerformValidation returns error only for internal logic errors (and the client may receive errors from gRPC in the event of a communication problem)."